__tests__/data/warehouse/events.js

```diff
@@ -0,0 +1,1489 @@
+const sampleEvents = {
+  track: {
+    input: {
+      destination: {
+        Config: {
+          restApiKey: "9432f11f70f8ce386f5110c8c924b3ec4f825256",
+          prefixProperties: true,
+          useNativeSDK: false
+        },
+        DestinationDefinition: {
+          DisplayName: "Braze",
+          ID: "1WhbSZ6uA3H5ChVifHpfL2H6sie",
+          Name: "BRAZE"
+        },
+        Enabled: true,
+        ID: "1WhcOCGgj9asZu850HvugU2C3Aq",
+        Name: "Braze",
+        Transformations: []
+      },
+      message: {
+        anonymousId: "e6ab2c5e-2cda-44a9-a962-e2f67df78bca",
+        channel: "web",
+        context: {
+          app: {
+            build: "1.0.0",
+            name: "RudderLabs JavaScript SDK",
+            namespace: "com.rudderlabs.javascript",
+            version: "1.0.5"
+          },
+          ip: "0.0.0.0",
+          library: {
+            name: "RudderLabs JavaScript SDK",
+            version: "1.0.5"
+          },
+          locale: "en-GB",
+          os: {
+            name: "",
+            version: ""
+          },
+          screen: {
+            density: 2
+          },
+          traits: {
+            city: "Disney",
+            country: "USA",
+            email: "mickey@disney.com",
+            firstname: "Mickey"
+          },
+          userAgent:
+            "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.117 Safari/537.36"
+        },
+        event: "button clicked",
+        integrations: {
+          All: true
+        },
+        messageId: "a6a0ad5a-bd26-4f19-8f75-38484e580fc7",
+        originalTimestamp: "2020-01-24T06:29:02.364Z",
+        properties: {
+          currency: "USD",
+          revenue: 50
+        },
+        receivedAt: "2020-01-24T11:59:02.403+05:30",
+        request_ip: "[::1]:53710",
+        sentAt: "2020-01-24T06:29:02.364Z",
+        timestamp: "2020-01-24T11:59:02.403+05:30",
+        type: "track",
+        userId: "9bb5d4c2-a7aa-4a36-9efb-dd2b1aec5d33"
+      },
+      request: {
+        query: {
+          whSchemaVersions: "v1"
```

Comment:
Do we need tests for whSchemaVersion `v0` ?

Reply:
will put a TODO for now